.packit.yaml

```diff
@@ -35,11 +35,11 @@ jobs:
     # do not get the version from a tag (git describe) but from the spec file
     get-current-version:
     - grep -oP '^Version:\s+\K\S+' packaging/convert2rhel.spec
-- job: tests
-  targets:
-    epel-7-x86_64:
-      distros: [centos-7, oraclelinux-7]
-    epel-8-x86_64:
-      distros: [centos-8.4, centos-8, oraclelinux-8.4, oraclelinux-8.6]
-  use_internal_tf: True
-  trigger: pull_request
+      #- job: tests
+      #  targets:
+      #    epel-7-x86_64:
+      #      distros: [centos-7, oraclelinux-7]
+      #    epel-8-x86_64:
+      #      distros: [centos-8.4, centos-8, oraclelinux-8.4, oraclelinux-8.6]
+      #  use_internal_tf: True
+      #  trigger: pull_request
```

Comment:
Just to be sure, we will still allow integration tests through `packit` for a while?

Reply:
That's right, let packit remain for now

Reply:
Thanks!

Reply:
Well, if we enable the packit AND use the GitHub actions, we will make a LOT more of DDoS to GitLab, since the packit integration will need to pull the environment files from GitLab defined in `plans/main.fmf`. 

@danmyway or @kokesak any you know of a way to conditionally pull that? I'm asking that because I think there is no difference from the `tmt` perspective in the execution from packit/github actions. Meaning that, if I uncomment the file in `plans/main.fmf` to pull the environment files, all jobs will pull that file.

Reply:
I don't think it's worth now spending time on investigating what kind of condition would limit using the environment file when tmt is triggered only through GitHub action and not from when triggered through Packit.

The new tmt version is to be deployed to the testing farm these days and then we won't care that much about the number of gitlab requests.

Reply:
Yeah, don't know much about conditioning that.
All I know of is the `trigger` context, but I don't think that it would be of any help.
Hopefully the TF workaround will solve the issue.

Reply:
Enabled again.